### DISCUSSION TITLE
Does support for Windows 8 end together with the support of Windows 7?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#2897] [Q&A Feedback] [answered]
Currently, Windows 7 and newer versions are supported. Windows XP isn't supported, as current versions of Sandboxie don't run on it, Windows Vista might still work, but isn't officially supported, even if it would still run.

At some point in the future, Windows 7 can't be supported anymore, for technical reasons. This means that at some point, there will be releases that can't be run on Windows 7. Will the same apply to Windows 8 and 8.1., at the same time?

If future releases of Sandboxie still work on Windows 8 or 8.1. (whereas they don't on Windows 7), will that be officially supported or is that just a 'happy little accident'? With other words, will the officially supported version of Windows be bumped to Windows 10, when support for Windows 7 ends?

**Comment by isaak654** [comment, marked as answer]
Qt 6 framework (on which Sandboxie Plus will be entirely based) has removed the support for Windows 7/8/8.1, but it looks like it could be unofficially extended in the 64-bit Plus builds for a while ([source](https://www.wilderssecurity.com/threads/sandboxie-resource-access-rules-v2-0.450239/page-2#post-3140722)).

In addition to Qt 6, Visual Studio 2022 has removed the support for 32-bit operating systems... I don't think 32-bit Plus builds will last long.

**Reply by bastik-1001** [reply]
I don't expect 8/8.1 not being supported anymore to have a huge impact on the user base. Likewise, Windows 7 is fading away judging [the collected stats from statcounter](https://gs.statcounter.com/os-version-market-share/windows/desktop/worldwide/).

Previously, I had assumed that 32bit versions of Windows were quite rare, nowadays, but apparently there are still downloads of Sandboxie releases for these. I don't know if a maintainer has some aggregated stats to see which packages are download how often for a longer period of time.
